cmd/agent/app/agent.go

```diff
@@ -205,6 +206,10 @@ func run(ctx context.Context, karmadaConfig karmadactl.KarmadaConfig, opts *opti
 		return err
 	}
 
+	if opts.EnableCertRotation {
```

Comment:
Maybe we don't need the option to control the controller, why don't we take this as a [disabled by default](https://github.com/karmada-io/karmada/blob/0c056c8aa68fb0a5d4892d6e32b99a0cfc40f158/cmd/agent/app/agent.go#L101) controller. How do you think?

Reply:
It is a good idea.